packages/sitecore-jss/src/graphql-request-client.ts

```diff
@@ -79,6 +94,17 @@ export class GraphQLRequestClient implements GraphQLClient {
     this.debug = clientConfig.debugger || debuggers.http;
   }
 
+  /**
+   * Factory method for creating a GraphQLRequestClient.
```

Comment:
Isn't this technically a factory for a client factory (returns `GraphQLRequestClientFactory` not `GraphQLRequestClient`)?
```suggestion
   * Factory method for creating a GraphQLRequestClientFactory.
```